sphinx_js/doclets.py

```diff
@@ -3,17 +3,20 @@
 from errno import ENOENT
 from json import load
 import os
+from parsimonious.exceptions import ParseError
 from os.path import abspath, relpath, splitext, sep
 import subprocess
 from tempfile import TemporaryFile, NamedTemporaryFile
 
 from six import string_types
 from sphinx.errors import SphinxError
+from sphinx.util import logging
 
 from .parsers import path_and_formal_params, PathVisitor
 from .suffix_tree import PathTaken, SuffixTree
 from .typedoc import parse_typedoc
 
+logger = logging.getLogger(__name__)
```

Comment:
Might as well import more deeply. "logging" doesn't add any useful context to "getLogger"; it's just more to read.

Reply:
Ah yes, I am not used to import deeply like that.